modules/computer.c

```diff
@@ -58,24 +62,6 @@ gchar *callback_env_var(void);
 gchar *callback_dev(void);
 #endif /* GLIB_CHECK_VERSION(2,14,0) */
 
-/* Scan callbacks */
-void scan_summary(gboolean reload);
-void scan_os(gboolean reload);
-void scan_security(gboolean reload);
-void scan_modules(gboolean reload);
-void scan_boots(gboolean reload);
-void scan_locales(gboolean reload);
-void scan_fs(gboolean reload);
-void scan_memory_usage(gboolean reload);
-void scan_display(gboolean reload);
-void scan_network(gboolean reload);
-void scan_users(gboolean reload);
-void scan_groups(gboolean reload);
-void scan_env_var(gboolean reload);
-#if GLIB_CHECK_VERSION(2,14,0)
-void scan_dev(gboolean reload);
-#endif /* GLIB_CHECK_VERSION(2,14,0) */
```

Comment:
Why these have been moved to the header file?  They're used only in this `.c`; it would be better if they were marked as `static` and the prototypes were kept here.

Reply:
> They're used only in this `.c`.

That is not entirely true, as `scan_os()` is called by [modules/computer/boots.c:31](https://github.com/lpereira/hardinfo/blob/3ff7143f29e332bee854c5f009cf3be550a29afc/modules/computer/boots.c#L31). The rest may be an overkill, indeed, — I just assumed those similarly looking functions might also be referenced elsewhere.